app/main/routes.py

```diff
@@ -169,20 +171,9 @@ def record():
     Returns:
         A rendered HTML page with record details.
     """
-    record_1 = {
-        "file_name": "file-b2.txt",
-        "status": "open",
-        "transferring_body": "Electoral Commission",
-        "consignment_id": "TDR-2023-H2QS",
-        "description": "Test description",
-        "date_last_modified": "2023-02-27",
-        "held_by": "The National Archives, Kew",
-        "legal_status": "Public Record(s)",
-        "rights_copyright": "Crown Copyright",
-        "language": "English",
-    }
-
-    return render_template("record.html", record=record_1)
+    file_metadata = get_file_metadata(record_id)
```

Comment:
1 thing to point out is around naming. We refer to "records" on the UI but the db refers to Files. Not a big deal but just wanted to point it out. I have kept it as is, but there are a couple of crossover points such as get_file_metadata accepts a record_id.